packages/cli/src/commands/dev.ts

```diff
@@ -78,12 +78,12 @@ export async function dev(file: string, { preact, publicDir }: { preact: boolean
     const asset = deployment.assets.find(asset => request.url === `/${asset}`);
 
     if (asset) {
-      const extension = path.extname(asset) as keyof typeof extensionToContentType;
+      const extension = path.extname(asset);
       console.log(chalk.black(`            Found asset: ${asset}`));
 
       reply
         .status(200)
-        .header('Content-Type', extensionToContentType[extension] || 'text/html')
+        .header('Content-Type', extensionToContentType(extension) || 'text/html')
```

Comment:
After looking at it, I think we should replace the fallback `text/html` to `text/plain` as it should be the default for textual files: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

Could you update this?

Reply:
okay, you are right. I'll update this~